docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: '3.2'
+services:
+  kbs:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile.grpc-as
+    # image: xynnn007/kbs:v0.5.0-rc2
+    command: [
+        "/usr/local/bin/kbs",
+        "--socket",
+        "0.0.0.0:8080",
+        "--config",
+        "/etc/kbs-config.json",
+        "--auth-public-key",
+        "/opt/confidential-containers/kbs/user-keys/public.pub",
+        "--insecure-http"
+      ]
+    restart: always # keep the server running
+    ports:
+      - "8080:8080"
+    volumes:
+      - ./data/kbs-storage:/opt/confidential-containers/kbs/repository:rw
+      - ./config/public.pub:/opt/confidential-containers/kbs/user-keys/public.pub
+      - ./config/kbs-config.json:/etc/kbs-config.json
+
+  as:
+    image: xynnn007/attestation-service:v0.5.0-rc0
+    ports:
+    - "50004:50004"
```

Comment:
Maybe we should add the mapping of the port of the PCCS (default is localhost:8081)? Otherwise the AS will not be able to access the PCCS on the host. In addition, we should also mount the configuration file `/etc/sgx_default_qcnl.conf` to the AS, and I think for this config file, we can also provide a default one in our `kbs/config/sgx_default_qcnl.conf`. What do you think?

Reply:
Yes. I think providing a `kbs/config/sgx_default_qcnl.conf` is a good choice, and let the user to edit the concrete PCCS url in the config.

Also, there is some bug in the AS side. After the PR is finished https://github.com/confidential-containers/attestation-service/pull/73,  I'll push more commits in this PR.